### PR TITLE
Adaptiveness

### LINKED ISSUE #3: Make UI adaptive
Telegraph currently looks a little… stretched in full screen.

Screenshot from 2023-03-23 22-39-52

We could e.g. take inspiration from [Dialect](https://apps.gnome.org/app/app.drey.Dialect/), which has a fully adaptive UI:

Screenshot from 2023-03-23 22-36-22

Screenshot from 2023-03-23 22-36-53

Screenshot from 2023-03-23 22-37-06

It doesn't necessarily look *perfect*, but it makes sense to stack the input and output next to each other when the window is in a landscape orientation.

### PR DESCRIPTION
**Pull request by gregorni**
Closes #3  

Still needs refinement, looks weird right now with button placement, but basic functionality is there.

### CODE REVIEW

**Comment by gregorni** [comment]
Yeah, I basically just copied Dialect's approach, but a squeezer seems the way to go.

**Comment by fkinoshita** [comment]
Works nicely! Thank you for your contribution :)

File: src/window.py [hunk 1]
```diff
@@ -96,6 +98,14 @@ def __init__(self, **kwargs):
 
         self.input_text_view.grab_focus()
```
Comment (fkinoshita) [inline src/window.py:100]: Could you please add one more line here?

File: data/ui/window.ui [hunk 2]
```diff
@@ -9,7 +9,7 @@
       <object class="AdwToastOverlay" id="toast_overlay">
 
         <property name="child">
-          <object class="GtkBox">
+          <object class="GtkBox" id="window_box">
```
Comment (fkinoshita) [inline data/ui/window.ui:12]: I believe you should change the child GtkBox and not this one

Comment (gregorni) [reply data/ui/window.ui]: Fixed now.

**Comment by fkinoshita** [comment]
Thanks, it's working as expected!

**Comment by gregorni** [comment]
Are there any plans to switch to `Adw.Squeezer` in the future?

**Comment by fkinoshita** [comment]
Not sure, I will test things out when I have time.

**Pull request merged**